root/static/scripts/edit/URLCleanup.js

```diff
@@ -862,13 +862,13 @@ validationRules[LINK_TYPES.songfacts.work] = function (url) {
 };
 
 // allow only Soundcloud pages with the Soundcloud rel
-validationRules[LINK_TYPES.soundcloud.artist] = function (url) {
-  return /soundcloud\.com\//.test(url);
-};
+function validateSoundCloud(url) {
+  return /soundcloud\.com\/(?!(search|tags)[\/?#])/.test(url);
+}
 
-validationRules[LINK_TYPES.soundcloud.label] = function (url) {
-  return /soundcloud\.com\//.test(url);
-};
+validationRules[LINK_TYPES.soundcloud.artist] = validateSoundCloud;
+validationRules[LINK_TYPES.soundcloud.label] = validateSoundCloud;
+validationRules[LINK_TYPES.soundcloud.series] = validateSoundCloud;
```

Comment:
Instead of assigning all of these directly (which is how we (I?) missed series), I think we should just do something like the wikipedia ones: https://github.com/metabrainz/musicbrainz-server/blob/1724da055c5d6485c0e7a9ff2de6c3ebf748810d/root/static/scripts/edit/URLCleanup.js#L828-L830